src/jobs/consumers/notifications/base-notification.consumer.ts

```diff
@@ -0,0 +1,39 @@
+import { Logger } from '@nestjs/common';
+import { Job } from 'bull';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Notification } from 'src/modules/notifications/entities/notification.entity';
+import { DeliveryStatus } from 'src/common/constants/notifications';
+import { NotificationsService } from 'src/modules/notifications/notifications.service';
+
+export abstract class BaseNotificationConsumer {
```

Comment:
Can we just keep it as notification consumer? Instead of base notification consumer?

Check this: https://youtu.be/-J3wNP6u5YU?t=278